### PR TITLE
paddle-bert

### PR DESCRIPTION
**Pull request by yaozhixin**

### CODE REVIEW

**Review by internetoftim — APPROVED** [review]

**Comment by payoto** [comment]
I can't see a requirements.txt in this example. Could you add a follow up PR with the way to set up the environment?

File: paddlepaddle/bert-base/README.md [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+
+cmake --build `pwd`/build --config Release --target paddle_python -j$(nproc)
+
+pip3.7 install -U build/python/dist/paddlepaddle-0.0.0-cp37-cp37m-linux_x86_64.whl
+```
+
+#### PaddleNLP
+
+```
+pip3.7 install jieba h5py colorlog colorama seqeval multiprocess numpy==1.19.2 paddlefsl==1.0.0 six==1.13.0 wandb
```
Comment (yaozhixin) [inline paddlepaddle/bert-base/README.md:136]: @payoto Hi, we have put the environment requirements in README here. The part of packages is depended on the installation of PaddleNLP (https://github.com/PaddlePaddle/PaddleNLP/blob/develop/requirements.txt), and the other part is depended on our Paddle-Bert project. So we put them together and install them in PaddleNLP stage. Is that ok?

Comment (internetoftim) [reply paddlepaddle/bert-base/README.md:136]: If this is in paddlepaddle repo, no need to add it. However, please explain this explanation in the readme -- on the quick setup guide, put it in the "Compile and installation" section

Comment (payoto) [reply paddlepaddle/bert-base/README.md:136]: Sorry @yaozhixin didn't mean to ping you on the weekend, I was just having a quick look at the repo and thought I'd mention it as we're looking to add CI so will need a bit of standardisation.

Comment (yaozhixin) [reply paddlepaddle/bert-base/README.md:136]: Hi, @payoto Please don't be sorry! It's so easy to check and fix! If there is any problem or suggestion, please let me know! Have a nice weekend!

Comment (yaozhixin) [reply paddlepaddle/bert-base/README.md:136]: Hi, @internetoftim I have committed a new [PR](https://github.com/graphcore/portfolio-examples/pull/9) in the morning, and I have committed an update based on the comments just now. Thanks! If there is any problem or suggestion, please let me know. Have a nice weekend!

**Pull request merged**